Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -35,7 +35,7 @@ Alternatively, if you are on Debian/Ubuntu, you can use the `install_prerequisit
 bash scripts/install_prerequisites.sh
 ```
 
-3. Make sure that you have **Python 2** installed, it is needed for the dev server to run. On Ubuntu 20 you can install it using `sudo apt install python2`.
+3. Make sure that you have **Python 2** installed, it is needed for the dev server to run. On Ubuntu 20 you can install it using `sudo apt install python2`. On Ubuntu 18 you can install it using `sudo apt install python-minimal`. If both of them do not work, try using `sudo apt install python2-minimal`.
```

Comment:
If both of them --> If both these commands

Reply:
okay I will fix it